### DISCUSSION TITLE
Which option should I choose?

### DISCUSSION THREAD

**Discussion by Ed1s0nZ** [Security-Onion-Solutions/securityonion#4977] [Unsupported Versions]
Which option should I choose?
I want to apply it to the real world of the enterprise, but I don't know which option to choose... Please give me some advice~
![image](https://user-images.githubusercontent.com/83812544/127437954-2df8e159-d08c-46ea-b253-95752dfc49d4.png)

**Comment by Az0res** [comment]
You can find the details for each option in the doc, at https://docs.securityonion.net/en/2.3/configuration.html#configuration.
Difficult to answer your question without more information.
